doc/plugins/authz-keycloak.md

```diff
@@ -40,14 +40,23 @@ For more information on Keycloak, refer to [Keycloak Authorization Docs](https:/
 
 | Name                    | Type          | Requirement | Default                                       | Valid                                           | Description                                                                                                                                                 |
 | ----------------------- | ------------- | ----------- | --------------------------------------------- | ----------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| token_endpoint          | string        | required    |                                               | [1, 4096]                                       | A OAuth2-compliant Token Endpoint that supports the `urn:ietf:params:oauth:grant-type:uma-ticket` grant type.                                               |
+| discovery               | string        | optional    |                                               | [1, 4096]                                       | URL to discovery document for Keycloak Authorization Services.                                                                                              |
```

Comment:
We should change the `[1, 4096]` to a valid example. It is the problem of original doc.

Reply:
done